app/models/provider.rb

```diff
@@ -108,6 +110,14 @@ def hei?
     accreditation_id.starts_with?("1")
   end
 
+  def performance_sign_offs
+    sign_offs.performance_profile
+  end
+
+  def performance_signed_off?
+    sign_offs.performance_profile.current_academic_cycle.exists?
```

Comment:
@tomtrentham 

![image](https://github.com/user-attachments/assets/7c1f31b3-7fbf-4860-bf74-aac69374604b)

Given that it refers to the `previous academic cycle`'s trainees but is signed off in the `current academic` cycle

Reply:
Yeah, we need to be super clear about whether the `academic_cycle` for the sign-off refers to the trainees' cycle or the cycle in which the sign-off happened!
I think the way we talk about the PP sign-off it makes sense to link it to the trainees' cycle. So it would be previous academic cycle.

Reply:
> Wouldn't the banner actually have the deadline in 2025 for the 2023 to 2024 performance profile, not 2024?

Umm just testing are you paying attention :sweat_smile: